templates/compte/delete_account_warning.html

```diff
@@ -0,0 +1,28 @@
+{% extends "editorial/base.html" %}
+
+{% load static %}
+
+{% block page_title %}Désactivation du compte{% endblock %}
+{% block robots %}noindex, nofollow{% endblock %}
+
+{% block editorial_content %}
+  <main role="main" id="content">
+    <div class="alert alert-danger">
+      Souhaitez vous confirmer la suppression de votre compte ? <br>
+      Toutes vos contributions sont conservées, en revanche, votre identifiant n'apparaîtra plus sur le site.
+    </div>
+
+    <div class="d-flex justify-content-between">
+      <span>
+        <a class="btn btn-sm btn btn-outline-info text-nowrap" href="{% url "delete_account" %}">
+         Retour
+        </a>
+      </span>
+      <span>
+        <a class="btn btn-sm btn-danger text-nowrap" href="{% url "delete_account" %}">
+         Confirmer la suppression
+        </a>
```

Comment:
This should definitely NOT be a link, rather a button which submits a form in POST, with csrf checks and so on. Also btw, the two "Retour" and "Confirmer la suppression" links both point to the same deletion URL, so that's why when clicking "Retour" the account is actually deleted.

Reply:
@celiacheff if that helps, it's basically exactly what has been done for erp deletion https://github.com/MTES-MCT/acceslibre/blob/master/erp/forms.py#L481-L493